cli/cobra.go

```diff
@@ -23,6 +23,7 @@ func SetupRootCommand(rootCmd *cobra.Command) {
 	rootCmd.SetHelpTemplate(helpTemplate)
 	rootCmd.SetFlagErrorFunc(FlagErrorFunc)
 	rootCmd.SetHelpCommand(helpCommand)
+	rootCmd.SetVersionTemplate("Docker version {{.Version}}\n")
```

Comment:
Wasn't sure if we needed this; without this, `docker` will be printed lowercase (i.e., name of the binary). If we think that's ok, I can remove this